synapse/config/oidc.py

```diff
@@ -27,7 +27,8 @@
 
 from ._base import Config, ConfigError, read_file
 
-DEFAULT_USER_MAPPING_PROVIDER = "synapse.handlers.oidc_handler.JinjaOidcMappingProvider"
+DEFAULT_USER_MAPPING_PROVIDER = "synapse.handlers.oidc.JinjaOidcMappingProvider"
+LEGACY_USER_MAPPING_PROVIDER = "synapse.handlers.oidc_handler.JinjaOidcMappingProvider"
```

Comment:
Can you add a quick comment to say why this is a thing, i.e. "we renamed the file..."

Reply:
Sure! 👍